### ISSUE TITLE
male vs female open flowers

### ISSUE THREAD

**Issue by edenny** [PlantPhenoOntology/ppo#73]
We have some datasets that distinguish between male flowers that are open and female flowers, so probably should add this to PPO

**Comment by ramonawalls** [comment]
Agree that we can add these terms. Will track their progress here.

**Comment by elhartley** [comment]
These are the PO terms I found for male and female flowers:

- Male flower: `staminate flower (PO:0025600)`
- Female flower: `pistillate flower (PO:0025599)`

Are these the terms that should be added?

**Comment by ramonawalls** [comment]
@elhartley yes!

**Comment by elhartley** [comment]
@ramonawalls Does this hierarchy look correct for the new terms (highlighted below: blue = added PO term, green = new PPO term)?

![Image](https://github.com/user-attachments/assets/1144ba17-ec0f-4ede-b603-54f692e9f5d0)

Let me know if there is anything I should change. You can view the details in the `73-male-female-open-flowers` branch.

**Comment by ramonawalls** [comment]
Yes! This looks good. Even though it's not part of the current release project, you can still make a PR for it and we can include it in the release.

**Comment by elhartley** [comment]
Fixed by #113

**Comment by elhartley** [comment]
@edenny You're right, I forgot to add the presence/present/absent terms for the open pistillate/staminate flower terms. I'll add them now.

**Comment by edenny** [comment]
Ok, great! I was also wondering if we should add a full set of traits for male and female flowers to mirror what we have for flowers. That would include non-senesced, open, unopened, pollen-releasing (for male only) and senesced. I'm not sure any of my target datasets have anything besides "open" for male/female flowers, but seems like a good idea to add it while we are thinking closely about it!

**Comment by edenny** [comment]
We discussed and think people will use these terms so decided to add them.

**Comment by ramonawalls** [comment]
The terms have been added in PR #113
